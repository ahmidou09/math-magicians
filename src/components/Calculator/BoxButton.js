import PropTypes from 'prop-types';

const BoxButton = ({ children }) => (
  <div className="buttonBox">
    {children}
  </div>
);
BoxButton.propTypes = {
  children: PropTypes.node.isRequired,
};

export default BoxButton;
