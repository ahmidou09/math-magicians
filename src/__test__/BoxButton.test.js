/* eslint-disable linebreak-style */
/* eslint-disable comma-dangle */
import React from 'react';
import { render } from '@testing-library/react';
import BoxButton from '../components/Calculator/BoxButton';

test('renders BoxButton component', () => {
  const { container } = render(
    <BoxButton>
      <button type="button">Button 1</button>
      <button type="button">Button 2</button>
    </BoxButton>
  );
  expect(container).toMatchSnapshot();
});
