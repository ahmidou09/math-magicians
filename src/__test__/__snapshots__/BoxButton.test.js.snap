// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders BoxButton component 1`] = `
<div>
  <div
    class="buttonBox"
  >
    <button
      type="button"
    >
      Button 1
    </button>
    <button
      type="button"
    >
      Button 2
    </button>
  </div>
</div>
`;
